solidity/contracts/mERC20.sol

```diff
@@ -32,19 +34,23 @@ abstract contract AbstractERC20Upgradeable is ERC20Upgradeable, ERC20BurnableUpg
 
     /// @dev Initializes the contract.
     function initialize(
-        string memory name,
-        string memory symbol,
-        address initialMinter
+        string memory _name,
+        string memory _symbol,
+        uint8 _decimalsArg,
+        address _minter
     ) public initializer {
-        __ERC20_init(name, symbol);
+        __ERC20_init(_name, _symbol);
+        __ERC20Burnable_init();
```

Comment:
Shouldn't we use the unchained version? `ERC20BurnableUpgradeable` inherits from `ERC20Upgradeable` and [based on the docs](https://docs.openzeppelin.com/contracts/4.x/upgradeable#multiple-inheritance) 

> __{ContractName}_init function embeds the linearized calls to all parent initializers.

And it is not something we would do ideally. Right now, `_init` does nothing so we are good. I could argue `_init_unchained` is more future-proof, given it should guarantee `__ERC20_init` will not get called twice. At the same time, using `_init_unchained` is not recommended but... what's the alternative?

Anyway, I am good leaving it as-is, just curious about your opinion.

Reply:
As you said, the general OZ recommendation is not using `unchained`. I'm just following it 🤷